data-serving/scripts/prune-uploads/prune_uploads.py

```diff
@@ -78,14 +78,15 @@ def find_acceptable_upload(
       (last acceptable upload, list of uploads to mark for deletion).
       If no acceptable upload is found, returns None
     """
-    uploads = source.get("uploads", [])
-    # skip accepted or rejected uploads
-    uploads = [u for u in uploads if "accepted" not in u]
-    if not uploads:
+    if not (uploads := source.get("uploads", [])):
         return None
 
     if source.get("hasStableIdentifiers", False):
-        return _ids(u for u in uploads if u['status'] != "IN_PROGRESS"), []
+        return _ids(
+            u for u in uploads
+            if u['status'] != "IN_PROGRESS"
```

Comment:
I find `dict.get('key', <default>)` better than `dict['key']`

Reply:
Normally I'd use that, but status is a required field in the schema so always present